src/OrderFormAcceptanceTests.Steps/Steps/Hooks.cs

```diff
@@ -73,5 +64,19 @@ public async Task AfterScenario()
                 await UsersHelper.Delete(test.IsapiConnectionString, user);
             }
         }
+
+        private static OrderingDbContext GetDbContext(string connectionString)
+        {
+            DbContextOptions<OrderingDbContext> options = new DbContextOptionsBuilder<OrderingDbContext>()
+                .UseSqlServer(connectionString, s =>
+                {
+                    s.EnableRetryOnFailure(5);
+                })
+                .Options;
+
+            OrderingDbContext dbContext = new(options);
+
+            return dbContext;
+        }
```

Comment:
Could you use a `Lazy<T>` here to avoid initializing the DB context multiple times?

If it's possibly to inject `UITest` into the constructor I think you could initialize a `Lazy<T>` in the constructor using a closure to capture the connection string.

Reply:
As discussed externally to this PR, this is not possible